backend/src/main/java/org/eclipse/tractusx/puris/backend/stock/logic/adapter/ItemStockSammMapper.java

```diff
@@ -121,13 +121,17 @@ public List<ReportedProductItemStock> itemStockSammToReportedProductItemStock(It
             log.warn("Direction should be INBOUND, aborting");
             return outputList;
         }
-        var mpr = mprService.findByPartnerAndPartnerCXNumber(partner, matNbrCatenaX);
 
-        if (mpr == null) {
+        Material material = materialService.findByMaterialNumberCx(matNbrCatenaX);
```

Comment:
As we already missed this: please add a comment on why we use this method in this scenario (same for other scenario) like "reported item stock with direction inbound means,  we are supplier \n As a supplier, we define the material number in the material, also for the partner. Thus, material Catena-X ID instead of mpr Catena-X ID is used". If one checks the service, one will know.

Reply:
Done